EventListener/AbstractConsumeEventListener.php

```diff
@@ -14,7 +14,7 @@
 use ONGR\ConnectionsBundle\Pipeline\Event\ItemPipelineEvent;
 
 /**
- * AbstractConsumeEventListener class.
+ * Handles basic item skipping when there is nothing to do on skip.
```

Comment:
This sentence becomes clear after reading it several times and also reading the code. Not sure if that's so great, but it's not that bad, either.